forta-tornado-cash-starter-kit/src/helper.js

```diff
@@ -36,6 +50,7 @@ const tornadoCashAddressesARBITRUM = [
 module.exports = {
   eventABI:
     "event Withdrawal(address to, bytes32 nullifierHash, address indexed relayer, uint256 fee)",
+  addressLimit: 10000,
```

Comment:
Since `tornadoCashAddressesETHER` has ~5x the number of smart contract addresses compared to other chains, could you increase the addressLimit to `50000` for just `tornadoCashAddressesETHER`.